conanfile.py

```diff
@@ -34,9 +31,20 @@ def set_version(self):
             cmake_file = load(self, os.path.join(self.recipe_folder, "CMakeLists.txt"))
             self.description = re.search(r"project\([^)]*DESCRIPTION\s+\"([^\"]+)\"[^)]*\)", cmake_file).group(1)
 
+    def layout(self):
+        cmake_layout(self)
```

Comment:
I didn't notice that you also did this in dice-hash, if I remember correctly this is wrong and should be removed. I think it causes issues downstream in consuming packages

Reply:
Please try it out downstream

Reply:
Its used in the official documentation: https://docs.conan.io/2/tutorial/creating_packages/other_types_of_packages/header_only_packages.html#header-only-library-with-tests
Also publish branch package does build the test_package, which tests if the package works correctly.

Reply:
Yeah I know, I vaguely recall that I ran into an issue anyway in the past

Reply:
Looks like everything works fine now: https://github.com/rdf4cpp/rdf4cpp/pull/282

Reply:
On my end the issue appears now

![grafik](https://github.com/dice-group/dice-sparse-map/assets/31625940/4b9c753f-fb26-44c1-b085-f73702e29b79)

Reply:
Ok, I think the issue that I remembered appears when you mix conan 1 and 2.. I think, maybe